clients/admin-ui/src/types/api/README.md

```diff
@@ -20,6 +20,12 @@ turbo run openapi:generate
 
 **The Fides server must be running on `localhost:8080`**. This will read the schema hosted at <http://localhost:8080/docs> and generate all TS files under `clients/admin-ui/src/types/api/`.
 
+You will also likely need to follow this by running `format`, in order to ensure the generated files are formatted correctly:
+
+```
+turbo run format
```

Comment:
just a process thing i noticed during my brief foray into FE dev work. it was noted that we probably should just bundle the `format` command into the generate command, but i wasn't immediately sure how to do that so i figured this was a fine first step.